gender_novels/analysis/gender_pronoun_freq_analysis.py

```diff
@@ -288,7 +288,7 @@ def get_mean(data_dict):
     return mean_dict
 
 
-if __name__ == "__main__":
+#if __name__ == "__main__":
```

Comment:
Probably better, if no function specified, run tests.
```
if __name__ == '__main__':
    from dh_testers.testRunner import main_test
    main_test()
```

Reply:
I think I fixed it